geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -1314,7 +1314,7 @@ public void addHostedLocators(InternalDistributedMember member, Collection<Strin
       this.hostedLocatorsAll = tmp;
 
       if (isSharedConfigurationEnabled) {
-        if (locators == null || locators.isEmpty()) {
+        if (locators.isEmpty()) {
```

Comment:
I think you can remove this if statement.  We've already checked if locators.isEmpty() earlier in the method.